src/renderers/shared/fiber/ReactDebugCurrentFiber.js

```diff
@@ -33,9 +33,13 @@ function getCurrentFiberOwnerName() : string | null {
     }
     switch (fiber.tag) {
       case IndeterminateComponent:
-      case FunctionalComponent:
       case ClassComponent:
         return getComponentName(fiber);
+      case FunctionalComponent:
```

Comment:
Why would you move them together with `HostComponent`? I'm not sure I'm following.